modulemd/modulemd-modulemetadata.c

```diff
@@ -400,331 +375,303 @@ modulemd_modulemetadata_get_summary (ModulemdModuleMetadata *self)
+
+  properties [PROP_NAME] =
+    g_param_spec_string ("name",
+                         "Module Name",
+                         "A string property representing the name of "
+                         "the module.",
+                         "",
+                         G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS);
+
+  properties [PROP_STREAM] =
+    g_param_spec_string ("stream",
+                         "Module Stream",
+                         "A string property representing the stream name "
+                         "of the module.",
+                         "",
+                         G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS);
+
+  properties [PROP_VERSION] =
+    g_param_spec_uint64 ("version",
+                         "Module Version",
+                         "An integer property representing the version of "
+                         "the module.",
+                         0,
+                         G_MAXUINT64,
+                         0,
+                         G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS);
+
+  properties [PROP_SUMMARY] =
+    g_param_spec_string ("summary",
+                         "Module Short Description",
+                         "A string property representing a short summary "
+                         "of the module.",
+                         "",
+                         G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS);
+
+  properties [PROP_DESCRIPTION] =
+    g_param_spec_string ("description",
+                         "Module Description",
+                         "A string property representing a detailed "
+                         "description of the module.",
+                         "",
+                         G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS);
+
+  properties [PROP_COMMUNITY] =
+    g_param_spec_string ("community",
+                         "Module Community",
+                         "A string property representing a link to the "
+                         "upstream community for this module.",
+                         "",
+                         G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS);
+
+  properties [PROP_DOCUMENTATION] =
+    g_param_spec_string ("documentation",
+                         "Module Documentation",
+                         "A string property representing a link to the "
+                         "upstream documentation for this module.",
+                         "",
+                         G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS);
+
+  properties[PROP_TRACKER] =
+    g_param_spec_string ("tracker",
+                         "Module Bug Tracker",
+                         "A string property representing a link to the "
+                         "upstream bug tracker for this module.",
+                         "",
+                         G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS);
+
+  /**
+   * ModulemdModuleMetadata:buildrequires: (type GLib.HashTable(utf8,utf8)) (transfer container)
```

Comment:
After considerable inspection, it turns out that this is the only line in this entire patch that has a functional effect. Why was this not part of the patch that "fixed" buildrequires at least? (Since that's where you change the transfer type as well).